arm-dependencies.sls

```diff
@@ -0,0 +1,89 @@
+{% from 'common/map.jinja' import config with context %}
+
+arm-dependencies:
+  pkg.installed:
+    - pkgs:
+      - g++-aarch64-linux-gnu
+      - g++-arm-linux-gnueabihf
+
+{% set path = 'https://servo-rust.s3.amazonaws.com/ARM' %}
+{% set targets = [{ 'name': 'arm-linux-gnueabihf',
```

Comment:
I'd put 'name' on the next line instead of the same line as the opening brace. (same below)